src/Microsoft.AspNetCore.Mvc.Razor/RazorPage.cs

```diff
@@ -223,9 +223,7 @@ public void StartTagHelperWritingScope(HtmlEncoder encoder)
 
             // We need to replace the ViewContext's Writer to ensure that all content (including content written
             // from HTML helpers) is redirected.
-            var buffer = new ViewBuffer(BufferScope, Path);
-            var writer = new HtmlContentWrapperTextWriter(buffer, ViewContext.Writer.Encoding);
-            ViewContext.Writer = writer;
+            ViewContext.Writer = new ViewBufferTextWriter(buffer, ViewContext.Writer.Encoding);
```

Comment:
Never mind. Couldn't find a case where the [chant](https://www.youtube.com/watch?v=e9mf3Bypyk8) isn't applicable.